about.html

```diff
@@ -0,0 +1,171 @@
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Titan+One&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="assets/CSS/about.css" />
+    <title>Nutrition</title>
+  </head>
+  <body>
+    <div class="container-fluid">
+      <div class="row">
+        <nav
+          class="navbar navbar-inverse text-white socialMediaIcons"
+          id="home"
+        >
+          <div class="icon-bar navbar-right">
+            <a href="#" class="facebook"><i class="fa fa-facebook"></i></a>
+            <a href="#" class="twitter"><i class="fa fa-twitter"></i></a>
+            <a href="#">My page</a>
+            <a href="#">Logout</a>
+          </div>
+        </nav>
+      </div>
+      <div class="row">
+        <nav class="navbar" id="navbar">
+          <div class="navbar-header">
+            <a class="navbar-brand" href="#">
+              <img src="assets/images/img4.png" alt="navbar logo imag" />
+            </a>
+          </div>
+          <ul class="nav navbar-nav navItem text-secondary" id="navMenu">
+            <li class="active"><a href="index.html">Home</a></li>
+            <li><a href="about.html">About</a></li>
+            <li><a href="#program">Program</a></li>
+            <li><a href="#sports">Sports</a></li>
+            <li><a href="#">Sponser</a></li>
+            <li><a href="#">Nutrition</a></li>
+          </ul>
+
+          <div class="humgurger">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+          </div>
+        </nav>
+      </div>
+      <div class="row introduction">
+        <p>"Welcome to Healthline Nutrition!"</p>
+        <h1 class="secondHeader">
+          The Definitive Guide<br />
+          to Healthy Eating in Real Life
+        </h1>
+
+        <p id="second-p">
+          healthy eating doesn’t have to be complicated. It’s entirely possible
+          to nourish your body while enjoying the foods you love.
+        </p>
+        <p class="third-p">
+          Please contact us per email for any further question about Nutrition
+          <span
+            ><a href="mailto:someone@example.com">nutirstion@gmai.com</a></span
+          >
+        </p>
+      </div>
+      <div class="row">
+        <section class="logo">
+          <h2 class="a">Creative Common<br />Nutrition 2016 Logo</h2>
+          <h2 class="b">Creative Common Nutrition 2016 Logo</h2>
+          <hr />
+          <p>
+            The logo of Nutirstion 2015 was decided <br />
+            through the logo competetion from 8.Jun to 7.July
+          </p>
+          <img src="/assets/images/img4.png" alt="logo" />
```

Comment:
Great job on fixing the issues mentioned by the previous reviewer, but kindly note that this image is not visible in your live version. Please fix that 🤗

![image](https://user-images.githubusercontent.com/73714615/190340739-d7220e05-dcd9-4ea4-9aa2-74d81ad648a8.png)

Reply:
Solved

Reply:
@awais-amjed  can you check again ?